### DISCUSSION TITLE
List Our Meme Coin on the Strict List

### DISCUSSION THREAD

**Discussion by TonyDream1** [jup-ag/token-list#3391] [Q&A]
We've got a dream: to see our meme coin listed on the Strict List of the Jupiter Token API. But we need your guidance to make it a reality.

How do we go about getting our meme coin listed on the Strict List? What steps should we take, and who should we reach out to? We're eager to follow the process and ensure that our coin meets all the necessary criteria.

Any advice or pointers you can provide would be greatly appreciated. We're ready to put in the work to make this happen, and we're counting on your expertise to help us navigate the journey.

Looking forward to your guidance, D.

**Comment by ghost** [comment]
https://station.jup.ag/docs/get-your-token-onto-jup
